.github/workflows/check-pr-title.yml

```diff
@@ -10,23 +10,13 @@ on:
 jobs:
   check-pr-title:
     runs-on: ubuntu-latest
-    permissions:
-      checks: write
-      pull-requests: read
-      contents: read
+    timeout-minutes: 5
     steps:
-      - uses: amannn/action-semantic-pull-request@v4
+      - name: Check out branch
+        uses: actions/checkout@v3
```

Comment:
Do we need to checkout the repo?

Reply:
For [parsing source](https://github.com/ivov/validate-n8n-pull-request-title/blob/a3616cc20f5a2bc6e6180270cac95253c1651836/src/parserContent.js) to extract all display names, based on which to validate node names.